### ISSUE TITLE
session sticky based on cookie or header hash strategy 

### ISSUE THREAD

**Issue by Yelijah** [traefik/traefik#9236]
### Welcome!

- [X] Yes, I've searched similar issues on [GitHub](https://github.com/traefik/traefik/issues) and didn't find any.
- [X] Yes, I've searched similar issues on the [Traefik community forum](https://community.traefik.io) and didn't find any.

### What did you expect to see?

Can traefik support session sticky based on cookie or header hash strategy which just like nginx's "hash  $cookie_xxx" ? 

In other words，I want that all requests with same header value or cookie value can be forward to one same server.

I couldn't find anything like it in doc or plugins.

**Comment by ddtmachado** [comment]
Duplicate of https://github.com/traefik/traefik/issues/1207
